### ISSUE TITLE
Neovim basic strategy to behave like Vim

### ISSUE THREAD

**Issue by dimanyc** [vim-test/vim-test#830]
First of all, thank you so much for this amazing tool! I've been using it for years!

The time has come for me to embrace the inevitable and migrate to nvim (better LSP option). 

I cannot stand using Nvim terminal emulator and would like to be able to just run tests in `bg` then `fg` back to Nvim, just like I did with Vim. 

I know this is probably more of a Neovim terminal question, but perhaps someone has figured this out.

I've tried creating a custom tmux strategies via vanilla tmux, VTR etc, but none of these options come close to the simplicity of just running tests in a job.

**Comment by codeinabox** [comment]
> I cannot stand using Nvim terminal emulator and would like to be able to just run tests in `bg` then `fg` back to Nvim, just like I did with Vim.

Thank you for all your kind words! Have you tried some of the other strategies, including Dispatch? How do you want the output of the test results displayed?

**Comment by dimanyc** [comment]
Yes, I tried Dispatch with tmux, tmux by itself, Vim Tmux Runner and Vimux. The closest I got is a strategy that opens a new tmux window called `test` and closes it on on any keypress:

```lua
  local function vimux_with_zoom_strategy(cmd)
    local window_name = "test"
    vim.g["VimuxRunnerType"] = "window"
    vim.g["VimuxRunnerName"] = window_name
    local full_cmd = cmd
        .. " || true; echo ''; echo 'Press any key to close'; read -n 1; tmux kill-window -t "
        .. window_name
    vim.fn["VimuxClearTerminalScreen"]()
    vim.fn["VimuxRunCommand"](full_cmd)
    vim.fn["VimuxZoomRunner"]()
  end

```
The problem with this strategy is that if I want to reference the test output I'd have to keep the "test" window open all the time. In the OG Vim basic strategy I can just toggle between foreground and background jobs.

**Comment by dimanyc** [comment]
here is what I'm living with for the past couple of months and it's pretty close. Requires tmux.

```lua
local function run_in_bg(cmd)
  -- Escape single quotes in the command
  cmd = cmd:gsub("'", "'\\''")

  -- Send the command to tmux buffer
  vim.fn.system(string.format("tmux set-buffer '%s'", cmd))

  -- Suspend Neovim
  vim.cmd("suspend")

  -- Paste the command, send Enter, and set up key binding to return to fg
  vim.fn.system(
    'tmux paste-buffer && tmux send-keys Enter && tmux bind-key -n any "send-keys fg Enter ; unbind-key -n any"'
  )
end

```

**Comment by aarushlohit** [comment]
can i work??? is this open

**Comment by tomatotomata** [comment]
I was thinking of adding a small Neovim strategy that preserves the familiar `bg`/`fg` flow, with a focused strategy spec and no changes to the existing Vim paths. I saw the tmux workaround in the thread, so I would keep this scoped to the strategy layer rather than trying to redesign terminal handling. Does that direction fit `vim-test`, or would you rather leave this to external configuration?
